client/src/containers/SignupForm.jsx

```diff
@@ -0,0 +1,124 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+const SignupForm = () => {
+  const [open, setOpen] = useState(false);
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setOpen(true);
+  };
+
+  const navigate = useNavigate();
+
+  const actionButtons = [
+    {
+      label: "Cancel",
+      onClick: () => {
+        setOpen(false);
+        navigate("/error");
+      },
+    },
+    {
+      primary: true,
+      label: "Accept and Register",
+      onClick: () => {
+        setOpen(false);
+        navigate("/success");
+      },
+    },
+  ];
+
+  return (
+    <Box
+      pad="medium"
+      direction="row"
+      flex="grow"
+      align="start"
+      justify="center"
+      responsive={false}
+    >
+      <Box direction="column" align="start" width="100%">
+        <Text size="xxlarge" weight="bold" color="text-strong">
+          Sign Up
+        </Text>
+        <Text size="small" color="text-strong">
+          Sign Up for GreenLake for Data Fabric beta access
+        </Text>
+
+        <form
+          onSubmit={handleSubmit}
+          data-testid="signupForm"
+          className="DfForm full"
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            width: "100%",
+          }}
+        >
+          <DfTextField
+            id="signUpName"
+            name="name"
+            label="Name"
+            required
+            // error="Name is required"
+          />
+          <DfTextField
+            id="signUpSurname"
+            name="Surname"
+            label="Surname"
+            required
+            type="Surname"
+            // error="Surname is required"
+          />
+          <DfTextField
+            id="signUpEmail"
+            name="Email"
+            label="Email"
+            required
+            type="Email"
+            error="Email is required"
+          />
+          <DfSelectField
+            id="signUpCountry"
+            name="Country"
+            options={["India", "USA", "Ukraine"]}
+            value={"India"}
+            label="Country"
+            required
+            error="Country is required"
+            onChange={({ option }) => {}}
+          />
+          <Box
+            direction="row"
+            margin={{ top: "medium" }}
+            background={{ color: "transparent", dark: false }}
+          >
+            <Button
+              type="submit"
+              primary
+              reverse
+              label="Read the HPE terms & conditions to Register"
+            />
+          </Box>
+        </form>
+      </Box>
+      <DfModal
+        id="termsAndConditionsModal"
+        title="Terms & Conditions"
+        open={open}
+        setOpen={setOpen}
+        onClose={() => {}}
+        actionButtons={actionButtons}
+        width="large"
+      >
+        <Box>
+          <Image fit="cover" src="/images/terms-and-conditions.png" />
```

Comment:
I am not able to see the full image and there is no scrollbar:
<img width="796" alt="image" src="https://user-images.githubusercontent.com/84544729/204342475-0dcaf53e-67fc-42b3-ad5e-c80338e56556.png">